examples/garrawarla/configs/modules.yaml

```diff
@@ -0,0 +1,290 @@
+modules:
+  enable::
```

Comment:
by looking at the Spack modules yaml, I think this enable::lmod should be deleted,
as the good one is further down, at the beginning of the `default:` section